composer.json

```diff
@@ -11,13 +11,16 @@
     }
   ],
   "require": {
-    "php": ">=7.1.0",
-    "nette/forms": "^v2.4.0",
-    "nette/http": "^v2.4.0",
-    "nette/utils": "^v2.4.0",
-    "nette/di": "^v2.4.0",
-    "nette/application": "^v2.4.0",
-    "latte/latte": "^v2.4.0"
+    "php": ">=7.2.0",
+    "ext-json": "*",
+    "latte/latte": "^2.10",
+    "nette/application": "^3.1",
+    "nette/bootstrap": "^3.1",
```

Comment:
Na co je tu nette/bootstrap?

Reply:
Byl tam hlavně kvůli zprovoznění Configurator v testech, rozšíření mi padalo, ale po úpravách testu podle https://github.com/nettrine/orm/blob/master/tests/toolkit/Container.php to už asi není potřeba.